.github/workflows/ci.yml

```diff
@@ -27,9 +28,10 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: "12.x"
-      - uses: subosito/flutter-action@123325ea84b0d4c11e96eb9899090b697bf51bed
+      - uses: subosito/flutter-action@v1
         with:
-          flutter-version: "2.0.3"
+          channel: 'stable'
```

Comment:
does this need to be explicit ?

Reply:
We could skip the stable mention actually. But the flutter version should be fix to avoid the dartdoc and dart-sdk versions to change